hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/LazyDefiningSpecificMemberPms.java

```diff
@@ -113,11 +97,26 @@ public void init(@Nonnull Context context) throws Exception {
             }
         }
         if (address == null && !addresses.contains(address)) {
-            throw new JetException("Cluster does not contain the required member: " + memberAddress);
+            throw new JetException("Cluster does not contain the required member");
         }
         return addr -> supplier;
     }
 
+    @Override
+    public int preferredLocalParallelism() {
+        return 1;
+    }
+
+    @Override
+    public boolean isReusable() {
+        return true;
```

Comment:
false, see the javadoc:
> Meta-supplier method invocation sequences for different concurrent job executions may be interleaved

but it would be best to make this PMS reusable. @burakgok